README.md

```diff
@@ -272,34 +276,6 @@ module.exports = {
 }
 ```
 
-
-#### Export Issue with Babel Versions Lower Than 7
```

Comment:
Could you rebase your PR (or just make a new one) that doesn't remove this?

Reply:
Oh! Ok! I'll create another one!